terraform/modules/vpc/main.tf

```diff
@@ -2,8 +2,9 @@
 data "aws_availability_zones" "available" {
   state = "available"
 }
-
-
+locals {
+  az_length = length(data.aws_availability_zones.available.names)
```

Comment:
This variable is not used. Please use this variable in the below code.

Reply:
previously i was a different.this will not be used

Reply:
@Nouman72884 You are already calculating length in the below code. Use this variable instead.

Reply:
i forget to remove this block before pushing